include/edlib/ChiLoc.h

```diff
@@ -93,6 +93,9 @@ namespace EDLib {
     /**
      * General class for local susceptibilities calculation.
      *
+     * By default, all local Green functions are computed. An array of orbital
```

Comment:
We basically have duplicated functionality here and in GreensFunction.h

I propose to move body of the constructor into base class function and, If you really want to have different off-diagonal orbitals for Chi and for G, pass group name as a parameter.

Reply:
I will try to.